oc/openshift_ops.py

```diff
@@ -184,3 +185,165 @@ def create_project(self, project):
             raise Exception(err)
 
         return _rc
+
+    def create_cephblockpool(
+        self,
+        namespace,
+        cephblockpool_name_prefix="autotests-cephblockpool",
+        **kwargs
+    ):
+        """
+        Creates cephblock pool
+
+        Args:
+            namespace (str): namespace to create cephblockpool
+            cephblockpool_name_prefix (str): prefix given to cephblockpool
+
+        kwargs:
```

Comment:
I would capitalize it as there is all capitalized, and if we will specify Kwargs in docs I would follow the same syntax as for Args, WDYT? If we decide to do it like this we can send another PR to have it in documentation.
Like:
```python
Kwargs:
    failureDomain (str): description
    size (int): description
```

Reply:
I think we should separate functions into OCS and OCP,  this(*create_cephblock*) qualifies as OCS function and as such should be moved there and use the OCS way of doing things.

Reply:
After discussing with Shylesh, moved the implementaion cephblock pool to RookCluster.